mln/models/module.py

```diff
@@ -102,6 +148,21 @@ def harvest(self):
 		self.is_setup = False
 		self.save()
 
+	def is_clickable(self): 
+		"""Returns True if the owner set up this module, or it doesn't need setup."""
+		return self.is_setup is None or self.is_setup is True 
+
+	def select_arcade_prize(self, user):
+		"""Select a random arcade prize for an arcade winner."""
+		chance = random.randrange(100)
+		sum = 0
+		for prize in self.guest_yields.all():
+			sum += prize.success_rate
+			if sum > chance:
+				add_inv_item(user, prize.item_id, prize.qty)
+				return prize
+		raise RuntimeError("Should have chosen a prize but didn't for some reason")
+

```

Comment:
I'd really appreciate if you didn't randomly move around methods for no reason during a PR, this is a pain to review as I can't tell whether you're just moving stuff around or actually changing some lines as well. I see no reason to move the methods around in this case in the first place, but if there is one, please do it in a separate PR in the future.